packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -364,6 +397,28 @@ export default {
             if (this.isDropdown && this.hasTrailingIcon) {
                 throw new TypeError(`Form field is set to have inputType="dropdown", but trailing icons can only be displayed one of the following inputTypes: "${VALID_TRAILING_ICON_INPUT_TYPES.join('", "')}"`);
             }
+
+            if (this.isAffixedField && !this.isAffixedType) {
+                const afixType = this.prefix ? 'prefix' : 'suffix';
+
+                throw new TypeError(`Form field is set to have a "${afixType}" and inputType="${this.inputType}", "${afixType}" is only available with one of the following inputTypes: "${VALID_AFFIXED_INPUT_TYPES.join('", "')}"`);
+            }
+
+            if (this.prefix && this.hasLeadingIcon) {
+                throw new TypeError('Form field is set to have a "prefix" and "leadingIcon", only one can be displayed');
+            }
+
+            if (this.suffix && this.hasTrailingIcon) {
+                throw new TypeError('Form field is set to have a "suffix" and "trailingIcon", only one can be displayed');
+            }
+
+            if (this.prefix.length > 3) {
+                throw new TypeError(`Form field is set to have a "prefix" of ${this.prefix.length} characters long. Prefix has a max length of 3 characters`);
+            }
+
+            if (this.suffix.length > 3) {
+                throw new TypeError(`Form field is set to have a "suffix" of ${this.suffix.length} characters long. Suffix has a max length of 3 characters`);
+            }
```

Comment:
As these are validating against a single prop value (rather than checking against two values) you can add this as a prop validator where the prop is initialised ([see this](https://vuejs.org/v2/guide/components-props.html#Prop-Validation))